src/api/options-state.md

```diff
@@ -285,14 +285,14 @@ Declara funciones de vigilancia que se invocan cuando cambian los datos.
 
   La opción `watch` espera un objeto donde las claves son las propiedades de la instancia del componente reactivo a vigilar (por ejemplo, las propiedades declaradas a través de `data` o `computed`) - y los valores son las correspondientes devoluciones de llamada. La devolución de llamada recibe el nuevo valor y el valor antiguo de la fuente vigilada.
 
-  Además de una propiedad a nivel de raíz, la clave también puede ser una simple ruta delimitada por puntos, por ejemplo `a.b.c`. Tenga en cuenta que este uso **no** admite expresiones complejas, sólo admite rutas delimitadas por puntos. Si necesita vigilar fuentes de datos complejas, utilice en su lugar la API imperativa de [`$watch()`](/api/component-instance.html#watch).
+  Además de una propiedad a nivel de raíz, la clave también puede ser una simple ruta delimitada por puntos, por ejemplo `a.b.c`. Tenga en cuenta que este uso **no** admite expresiones complejas, sólo admite rutas delimitadas por puntos. Si necesita vigilar fuentes de datos complejas, utilice en su lugar la API imperativa de [`$watch()`](/api/component-instance#watch).
 
   El valor también puede ser una cadena con el nombre de un método (declarado mediante `methods`), o un objeto que contenga opciones adicionales. Si se utiliza la sintaxis de objeto, la llamada de retorno debe declararse en el campo `handler`. Las opciones adicionales incluyen:
 
   - **`immediate`**: activa la llamada de retorno inmediatamente después de la creación del observador. El valor antiguo será `undefined` en la primera llamada.
-  - **`deep`**: forza el recorrido profundo de la fuente si es un objeto o un array, para que la llamada de retorno se dispare en las mutaciones profundas. Ver [Watchers Profundos](/guide/essentials/watchers.html#watchers-profundos).
-  - **`flush`**: ajusta el tiempo de sincronización de la llamada de retorno. Ver [Temporización del Flujo del Callback](/guide/essentials/watchers.html#temporizacion-del-flujo-del-callback) y [`watchEffect()`](/api/reactivity-core.html#watcheffect).
-  - **`onTrack / onTrigger`**: depura las dependencias del observador. Ver [Depuración del Watcher](/guide/extras/reactivity-in-depth.html#depuracion-del-watcher).
+  - **`deep`**: forza el recorrido profundo de la fuente si es un objeto o un array, para que la llamada de retorno se dispare en las mutaciones profundas. Ver [Watchers Profundos](/guide/essentials/watchers#watchers-profundos).
+  - **`flush`**: ajusta el tiempo de sincronización de la llamada de retorno. Ver [Temporización del Flujo del Callback](/guide/essentials/watchers#temporizacion-del-flujo-del-callback) y [`watchEffect()`](/api/reactivity-core#watcheffect).
+  - **`onTrack / onTrigger`**: depura las dependencias del observador. Ver [Depuración del Watcher](/guide/extras/reactivity-in-depth#depuracion-del-watcher).
```

Comment:
change to 
#deep-watchers
#callback-flush-timing
#watcher-debugging
respectively